Jenkinsfile

```diff
@@ -51,15 +48,14 @@ pipeline {
             }
         }
 
-        stage('Run install') {
-            steps {
-                sh './mvnw org.jacoco:jacoco-maven-plugin:0.8.3:prepare-agent clean install org.jacoco:jacoco-maven-plugin:0.8.3:report coveralls:report --quiet --batch-mode -nsu'
+        stage('Install & Test on JDK 1.8') {
+            tools {
+                jdk 'JDK 1.8 (latest)'
```

Comment:
Here we need another stage to run test on OpenJDK, but since I don't know the OpenJDK tools name in Jenkins, other committer or PMC maybe can see it in the `Config Jenkins` menu and add it here

Reply:
We have the some privilege as you too.